modules/planning/tasks/optimizers/poly_vt_speed/BUILD

```diff
@@ -1,124 +1,124 @@
-load("//tools:cpplint.bzl", "cpplint")
+# load("//tools:cpplint.bzl", "cpplint")
 
-package(default_visibility = ["//visibility:public"])
+# package(default_visibility = ["//visibility:public"])
 
-cc_library(
-    name = "piecewise_poly_speed_curve",
-    srcs = [
-        "piecewise_poly_speed_curve.cc",
-    ],
-    hdrs = [
-        "piecewise_poly_speed_curve.h",
-    ],
-    copts = ["-DMODULE_NAME=\\\"planning\\\""],
-    deps = [
-        "//modules/common/proto:pnc_point_proto",
-        "//modules/planning/math/curve1d:cubic_polynomial_curve1d",
-        "//modules/planning/math/curve1d:quartic_polynomial_curve1d",
-        "//modules/planning/math/curve1d:quintic_polynomial_curve1d",
-    ],
-)
+# cc_library(
```

Comment:
How about the corresponding source files? Are they going to be removed as well?

Reply:
I will deprecate poly vt in next commit if everyone agrees @YajiaZhang @yifeijiang @ycool